wordpress-mobile/gutenberg-mobile
# [Low pri] Fix build for node v12.4.0

Fixes build errors on node v12.4.0

To test:
- `nvm install --latest-npm`
- `yarn start:reset`
- Check that you can build and run the app

Update release notes:
